src/instructlab/model/train.py

```diff
@@ -15,6 +15,9 @@
     TrainingArgs,
     run_training,
 )
+
+from instructlab.train.run_e2e_training import run_e2e_training
```

Comment:
implementing this in parallel, just waiting on `instructlab.eval` to publish their recent merges.

Reply:
so, this will in the training lib call the eval lib? or is the idea that we will call eval in the cli and call training over and over?

IMO, I am pro keeping all the orchestration in the CLI not the libraries

Reply:
I'd like to build orchestration below the level of the CLI since the training/eval libraries are sharing their interfaces like any other library.... and because I think it's janky to subprocess like 25 different things during training. But there's also the argument to be made that calling the CLI recursively also self-tests, which is useful. idk, how about a prototype and we see if it's reasonable?